gst/tensor_filter/tensor_filter.c

```diff
@@ -594,6 +600,10 @@ gst_tensor_filter_set_property (GObject * object, guint prop_id,
         g_strfreev (str_names);
       }
       break;
+    case PROP_NNAPI:
+      prop->use_nnapi = g_value_get_boolean (value);
+      silent_debug ("USE_NNAPI = %s\n", prop->use_nnapi ? "TRUE" : "FALSE");
+      break;
```

Comment:
I think we could give proper message to user if the framework does not support nnapi.

Reply:
User can see "USE_NNAPI=FALSE" message in default if use-nnapi option isn't passed to tensor filter. Do you mean to give more specific message? I.e., "tensorflow lite will use CPU fallback instead of NNAPI path"